servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/NettyHttpServer.java

```diff
@@ -260,9 +258,8 @@ public String toString() {
             this.headersFactory = headersFactory;
             executionContext = new DefaultHttpExecutionContext(connection.executionContext().bufferAllocator(),
                     connection.executionContext().ioExecutor(), connection.executionContext().executor(),
-                    strategy);
+                    HttpExecutionStrategies.noOffloadsStrategy());
```

Comment:
this looks suspicious. is there a chance we may return an unexpected value for `executionStrategy ()`? do we need `HttpExecutionStrategy executionStrategy()` on `HttpExecutionContext`?

Reply:
The strategy for the server context is always now noOffloads since offloading is handled by the OffloadingFilter. It does seem that it could possibly be removed.

Reply:
we should have a plan for offloading methods that may call back user code on the IOThread (e.g. `closeAsync()`, `closeAsyncGracefully()`, `onClose()`) for the `HttpServiceContext` passed to the user's service (and filters). It maybe nice to defer this until the offloading filters bcz we have some internal use cases where we don't want offloading to be applied (like `ChannelSet` that we disabled offloading for).

Reply:
Acknowledged.